sdk/identity/Azure.Identity/src/AadIdentityClient.cs

```diff
@@ -176,7 +176,7 @@ private Request CreateClientSecretAuthRequest(string tenantId, string clientId,
 
             request.Uri.Reset(_options.AuthorityHost);
 
-            request.Uri.AppendPath(tenantId);
+            request.Uri.AppendPath(tenantId, true);
```

Comment:
Any reason not to use named parameters consistently here (or not use them all over)?